src/util.jl

```diff
@@ -361,11 +361,11 @@ julia> finddelay([1, 2, 3], [0, 0, 1, 2, 3])
 ```
 """
 function finddelay(x::AbstractVector{<: Real}, y::AbstractVector{<: Real})
-    s = xcorr(y, x)
+    s = xcorr(y, x, padmode=:none)
```

Comment:
I think the safest thing would be to use `padmode=:longest`, which maintains the behavior that this function assumed.

Reply:
Well, it shifts the reference point, but I actually find this version a bit simpler.

Reply:
Just noticed you were taking the length of `x` below, not `s`. Yeah that is simpler!